ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -70,6 +76,23 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-with-aide-0.18
```

Comment:
We have the ability to say this is an optional test, right?

Reply:
I think we should be able to flag this as optional using:

```yaml
- as: e2e-aws-with-aide-0.18
  optional: true
```

I used a similar approach in https://github.com/openshift/release/pull/53502/files#diff-2d7db15a270722a31a7790ec2dbe40276c404f6b6443df19be407bcc90b5242eR82